code/game/machinery/doppler_array.dm

```diff
@@ -37,9 +37,6 @@ GLOBAL_LIST_EMPTY(doppler_arrays)
 	logged_explosions.Cut()
 	return ..()
 
-/obj/machinery/doppler_array/process()
-	return PROCESS_KILL
```

Comment:
Put this back. The whole point of this is to remove the excess processing overhead of processing an object with no `process()` call

Reply:
![image](https://user-images.githubusercontent.com/15887760/96769640-b2015a80-13df-11eb-952c-cc92c92cb403.png)
It's dupe code